### PR TITLE
added myself

### PR DESCRIPTION
**Pull request by RedTechEngineer**
:)

### CODE REVIEW

File: assets/capes.json [hunk 1]
```diff
@@ -335,4 +335,8 @@
     "uuid": "61f845f0-2502-4459-9fb4-e46d2600281c",
     "url": "https://raw.githubusercontent.com/kami-blue/assets/assets/assets/capes/cape13.png"
   }
```
Comment (5HT2) [inline assets/capes.json:337]: you're missing a comma here >:(

run it through something like https://jsonlint.com lol

File: assets/donators.json [hunk 2]
```diff
@@ -359,4 +359,8 @@
     "uuid": "61f845f0-2502-4459-9fb4-e46d2600281c",
     "type": "5"
   }
```
Comment (5HT2) [inline assets/donators.json:361]: comma here as well

File: assets/donators.json [hunk 3]
```diff
@@ -359,4 +359,8 @@
     "uuid": "61f845f0-2502-4459-9fb4-e46d2600281c",
     "type": "5"
```
Comment (5HT2) [inline assets/donators.json:360]: what did u contribute, iirc you're just a donator lol

**Review by 5HT2 — CHANGES_REQUESTED** [review]

**Comment by 5HT2** [comment]
aaa u could've just edited your fork with the new changes, no need for a new pull lol

**Comment by olliem5** [comment]
Technically, he contributed the code for the cape 😎

**Pull request closed without merging**